src-ghc/Pact/Main.hs

```diff
@@ -191,7 +191,7 @@ main = O.execParser argParser >>= \as -> case as of
     runScript dolog fp coverage = do
       (ref,coverF) <- handleCov coverage
       r <- execScriptF dolog fp coverF
-      void $ traverse writeCovReport ref
+      withCurrentDirectory (takeDirectory fp) $ void $ traverse writeCovReport ref
```

Comment:
optionally here you can use `traverse_` instead of `void $ traverse ...`

Reply:
used `mapM_` to avoid an import 👍